### ISSUE TITLE
checkpoint parity has no reachable tolerance for a quantized artifact

### ISSUE THREAD

**Issue by avifenesh** [avifenesh/memra#276]
`model verify checkpoint` compares memra's reference executor against an HF oracle, both declaring `numeric_class: source-weights-float32-accumulation`. For a DENSE checkpoint that is apples to apples. For a QUANTIZED one it is not, and the gate cannot pass.

Ran it for the first time on DictaLM-3.0-24B NVFP4 (`tiyuvta/DictaLM-3.0-24B-NVFP4`, llama_dense pack), oracle = the same artifact loaded through compressed-tensors with `run_compressed=False` and cast to f32 on CPU:

```
parity failed at token 0: abs=0.03934717 exceeds atol+rtol*abs(reference)=0.032463048
observed max_abs=0.10989499 at token 82580, max_rel=4.8032722
```

The model is not wrong. Decoding both logit vectors (131072 entries each):

```
argmax  oracle=1032 (6.3399)   native=1032 (6.3523)   MATCH
top5    oracle [1032, 1267, 1603, 1010, 3922]
        native [1032, 1267, 1603, 1010, 3922]        identical, same order
max_abs=0.10989  median_abs=0.020205  mean_abs=0.021794
```

Uniform ~0.02 noise on logits of magnitude ~6.3, with argmax and the whole top-5 preserved. That is two FP4 dequantization paths rounding differently, not a different execution program. The pack asks for `max_abs: 0.005`, which neither side can meet because neither is the other's reference: compressed-tensors dequantizes one way, memra another, and the gate then calls the difference a failure.

Two ways out, both design calls rather than fixes I want to make unilaterally:

1. the oracle consumes memra's OWN dequantized weights, so the comparison isolates the executor and the tolerance stays tight; or
2. `CheckpointParityGate` grows a quantized-artifact tolerance (with `require_argmax` doing the real work, since it is the part that is actually meaningful here).

Until one exists, no quantized artifact can reach `NativeQualified` through this gate, and DictaLM stays at `NativeReference` on that basis rather than on a missing run.

Receipts on the qualification box (vast 50033666): `/root/receipts-parity/` holds `hf-oracle.tsv`, `native-oracle.tsv`, the tensor census and the tokenizer contract. Tensor contract and tokenizer both PASSED; only the logit comparison is red.

https://claude.ai/code/session_01TrK6b3AFuBpsTQTvMLToor
